### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by BernadetteDiemer**
The links from the profile page to the events etc. are not yet implemented, also the Reviews aren't here yet. Otherwise it is functional and frontend-wise ok. 
Bildschirmfoto 2021-11-18 um 16 12 26
Bildschirmfoto 2021-11-18 um 16 13 03

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                  </div>
+                  <i class="fas fa-glass-cheers"></i>
+                </div>
+              </div>
+            <% end %>
+          <% end %>
+        <% end %>
+    </div>
+  </div>
+<% end %>
```
Comment (Lindadao92) [inline app/views/users/show.html.erb:99]: let's add some <br> somewhere here because of the menubar :-D

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      <div class="overflow-auto mb-10 md:mb-20">
+        <div class="d-flex flex-row sm:space-y-0">
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Events you created
+            </h3>
+          </div>
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem" style="padding: 0px 10px;">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Events you will attend
```
Comment (Lindadao92) [inline app/views/users/show.html.erb:39]: Lets link it to bookings#index already

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,99 @@
+<%= render 'shared/menubar' %>
+<%= render 'shared/navbar' %>
+
+<% if @user == current_user %>
+  <div class="container text-center justify-content-center">
+    <h1>Hi, <%= current_user.first_name %>!</h1>
```
Comment (Lindadao92) [inline app/views/users/show.html.erb:6]: remove the coma

**Review by Lindadao92 — CHANGES_REQUESTED** [review]
Looks great!!

File: app/views/users/show.html.erb (hunk 1 above)
Comment (BernadetteDiemer) [reply app/views/users/show.html.erb:99]: add some space?

Comment (BernadetteDiemer) [reply app/views/users/show.html.erb:99]: ach! '<br>'? :D

Comment (BernadetteDiemer) [reply app/views/users/show.html.erb:99]: ach! `<br>`

File: app/views/users/show.html.erb (hunk 2 above)
Comment (BernadetteDiemer) [reply app/views/users/show.html.erb:39]: `<%= link_to bookings_path do %>
            <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem" style="padding: 0px 10px;">
              <h3 class="profile-card pink mt-3 md:mt-0 i9prqz1">Events you will attend
              </h3>
            </div>
          <% end %>`

File: app/views/users/show.html.erb (hunk 3 above)
Comment (BernadetteDiemer) [reply app/views/users/show.html.erb:6]: Virna will be happy about this, she wanted to remove the comma way earlier haha I thought it looked good, but I am overruled now :D

**Pull request merged**